Iva-Georgieva/Exercise02-Basic/Basics01/BASICS01-007.js

```diff
@@ -0,0 +1,14 @@
+//BASICS01-007: Declare ten variables, assign the numbers from 1 to 10 and print the first three odd numbers on the console.
+
+let x1 = 1;
+let x2 = 2;
+let x3 = 3;
+let x4 = 4;
+let x5 = 5;
+let x6 = 6;
+let x7 = 7;
+let x8 = 8;
+let x9 = 9;
+let x10 = 10;
+
+console.log(x1 , x3 , x5);
```

Comment:
That's fine, you can just improve the code by following the name convention, there are too much intervals in the console.log()

```js
console.log(x1, x3, x5);
```